sphinx/registry.py

```diff
@@ -143,14 +147,14 @@ def preload_builder(self, app: "Sphinx", name: str) -> None:
             return
 
         if name not in self.builders:
-            entry_points = iter_entry_points('sphinx.builders', name)
+            builder_eps = entry_points(group='sphinx.builders')
```

Comment:
What does `eps` means? If you mean abbr of "EntryPointS", it would be better to use `builder_entry_points` instead. It's enough short and simple.

Reply:
I agree with the sentiment — clarity is important.

Reply:
Sure, done - and the same for the other file I've modified, of course.